web-app/src/components/common_styled/InputText.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export default styled.input`
+  font-family: inherit;
+  font-size: 100%;
+  line-height: 1.15;
+  margin: 10px 0px 10px 0px;
+  height: 34px;
+  width: 100%;
```

Comment:
Again is width set needed are we using bootstrap @William-Unosquare ?

Reply:
Bootstrap isn’t used in the project.